vizro-core/tests/unit/vizro/models/_controls/test_filter.py

```diff
@@ -427,6 +428,7 @@ def test_filter_build(self, test_column, test_selector):
         filter = vm.Filter(column=test_column, selector=test_selector)
         model_manager["test_page"].controls = [filter]
         filter.pre_build()
-        result = str(filter.build())
-        expected = str(test_selector.build())
-        assert result == expected
+        result = filter.build()
```

Comment:
Do we do this `str` conversion in any other tests at the moment? Because it's definitely not a good way to run tests. So we should make sure we remove all of them if it appears elsewhere...

Reply:
That was the last place where we had the object to string conversion. 😄